actions/workflow-invoker/src/types/data.rs

```diff
@@ -7,3 +7,20 @@ pub struct DbDatas {
     pub validator: String,
     pub key: String,
 }
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct Topic {
+    #[serde(rename = "_id")]
+    pub id: String,
+    #[serde(rename = "_rev")]
+    pub rev: String,
+    pub data: Vec<UserData>,
+}
+
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Default)]
+pub struct UserData {
+    pub user_id: String,
+    pub status: String,
```

Comment:
```suggestion
    pub status: String,
```

please use enum for this